pkg/cephmgr/mon/config.go

```diff
@@ -303,6 +304,7 @@ func CreateDefaultCephConfig(context *clusterd.Context, cluster *ClusterInfo, ru
 			OsdObjectStore:         store,
 			RbdDefaultFeatures:     3,
 			CrushtoolPath:          "",
+			FatalSignalHandlers:    "false",
```

Comment:
ceph does some other things besides just print the call stack in it's fatal signal handlers, such as print out all the recent events and the ceph version number/hash.  do we lose that functionality?  is that functionality useful?